src/modules/ConfigurationModule.js

```diff
@@ -22,4 +22,11 @@ module.exports = class ConfigurationModule extends Module {
   async setLanguage (_guild, language) {
     await this._guilds.update(_guild, { language })
   }
+
+  async setDeleteUserMessage (_guild, state) {
+    const guild = await this._guilds.get(_guild, 'deleteUserMessage')
+    if (guild.deleteUserMessage === state) throw new Error('SAME_STATE')
+    guild.deleteUserMessage = state
+    await guild.save()
```

Comment:
replace l.29 and l.30 with `await this._guilds.update(_guild, { deleteUserMessage: state })`

Reply:
yea, PORQUE TU NÃO REVISOU ESSA PORRA NO JOINLOCK, mas eu vou mudar, relaxa